test-quickstart-template.groovy

```diff
@@ -4,50 +4,42 @@ node('quickstart-template') {
-        def isJenkins = env.JOB_BASE_NAME.contains("jenkins")
+        def run_tests = true
+        if (env.JOB_BASE_NAME == "101-jenkins" || env.JOB_BASE_NAME == "201-jenkins-acr" || env.JOB_BASE_NAME == "azure-jenkins") {
+            // Any test will require an ssh tunnel be setup, but we're waiting for this bug to be fixed for ssh to work with the password-based templates: https://github.com/Azure/azure-cli/issues/2616
+            run_tests = false
+        }
 
-        def venv_name= env.JOB_BASE_NAME + '-venv'
-        def activate_venv = '. "' + env.WORKSPACE + '/' + venv_name + '/bin/activate";'
+        def run_basic_spinnaker_test = env.JOB_BASE_NAME.contains("spinnaker")
+        def run_spinnaker_k8s_test = env.JOB_BASE_NAME.contains("spinnaker") && env.JOB_BASE_NAME.contains("k8s")
 
         def scenario_name = "qstest" + UUID.randomUUID().toString().replaceAll("-", "")
 
-        stage('Init') {
+        stage('Deploy Quickstart Template') {
             dir('devopsci') {
                 checkout scm
-            }
-            if (isKubernetes && isSpinnaker) {
-                sh 'virtualenv ' + venv_name
 
-                dir('spinnaker') {
-                    git 'https://github.com/spinnaker/spinnaker.git'
-                }
-                dir('citestpackage') {
-                    // Eventually this repo will be its own python package and we won't have to install it separately
-                    git 'https://github.com/google/citest.git'
-                    sh activate_venv + 'pip install -r requirements.txt'
+                def script_path = 'scripts/deploy-quickstart-template.sh'
+                sh 'sudo chmod +x ' + script_path
+                withCredentials([usernamePassword(credentialsId: 'AzDevOpsTestingSP', passwordVariable: 'app_key', usernameVariable: 'app_id')]) {
+                    withCredentials([string(credentialsId: 'QsStorageConnectionString', variable: 'connection_string')]) {
+                        sh script_path + ' -tn ' + env.JOB_BASE_NAME + ' -sn ' + scenario_name + ' -ai ' + env.app_id + ' -ak ' + env.app_key + ' -cs "' + env.connection_string + '"'
+                    }
                 }
             }
         }
 
-        stage('Deploy Quickstart Template') {
-            def script_path = 'devopsci/scripts/deploy-quickstart-template.sh'
-            sh 'sudo chmod +x ' + script_path
-            withCredentials([usernamePassword(credentialsId: 'AzDevOpsTestingSP', passwordVariable: 'app_key', usernameVariable: 'app_id')]) {
-                withCredentials([string(credentialsId: 'QsStorageConnectionString', variable: 'connection_string')]) {
-                    sh script_path + ' -tn ' + env.JOB_BASE_NAME + ' -sn ' + scenario_name + ' -ai ' + env.app_id + ' -ak ' + env.app_key + ' -cs "' + env.connection_string + '"'
+        if (run_tests) {
+            sh 'ssh -F ' + scenario_name + '/ssh_config -f -N tunnel-start'
```

Comment:
I believe so. I'm running Jenkins jobs on my own instance right now to verify these changes and they're not to this point yet